ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/Smoke_Test.java

```diff
@@ -504,6 +506,33 @@ public void testRecoverWriteJob() throws SignatureException, IOException, XmlPro
     }
 
     @Test
+    public void crc32() throws IOException, SignatureException, XmlProcessingException, URISyntaxException {
```

Comment:
ehh, this test name could be a little more verbose, but I suppose it gets the idea across.